src/main/java/com/openmrs/migrator/core/services/impl/SettingsServiceImpl.java

```diff
@@ -0,0 +1,115 @@
+package com.openmrs.migrator.core.services.impl;
+
+import com.ibatis.common.jdbc.ScriptRunner;
+import com.openmrs.migrator.core.services.SettingsService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.InputStream;
+import java.io.Reader;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import org.apache.commons.lang.StringUtils;
+import org.pentaho.di.core.KettleEnvironment;
+import org.pentaho.di.core.util.EnvUtil;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SettingsServiceImpl implements SettingsService {
+
+  public void initializeKettleEnvironment(boolean testDbConnection) throws Exception {
```

Comment:
better to throw a specific type of exception to help identify source of bugs when they arise. Generally not a good idea to use base exception class